src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector (it impacts performance). It's better to set styles only for elements that require them. Consider removing this rule or replacing it with more specific selectors if necessary.